datarepo-clienttests/gradle/wrapper/gradle-wrapper.properties

```
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```

```suggestion
distributionUrl=https://services.gradle.org/distributions/gradle-6.1.1-bin.zip
```

Should the backslash be in the url?

---

I think so. That's how it is in their docs (https://docs.gradle.org/current/userguide/gradle_wrapper.html) and in our datarepo gradle-wrapper.properties file.